docs/manage/partitions/flex/faq.md

```diff
@@ -77,11 +79,13 @@ We recommend you configure partitions to have less than 5 TB per day flowing int
 * Reduces the cost of each query.
 * Accelerates queries, as less data has to be scanned.
 
-## How do I optimize my query using default scope?
+## How can I optimize my query using default scope?
 
 Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, if a query needs to run through 10 partitions, which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and reduce scan cost. You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating/updating your partition. Let's say that out of 10 partitions, you excluded two partitions. Now, when you run a query that does not have `_index` / `_view` term referenced in the query, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
 
-When partitions are marked as included and `_index`/`_view` is not referenced in the query, all the included partitions will be considered for the search query by default. Default scope can also come into use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
+You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating or updating the partition. If, out of 10 partitions, you exclude two partitions, any query that does not reference `_index` or `_view` will only consider the included partitions. This reduces the amount of data scanned and lowers the cost.
```

Comment:
@kimsauce, this paragraph repeats the last half of the previous paragraph.

Reply:
YIKES! Good catch. This was a merge conflict error on my part. I'll fix.

Reply:
@kimsauce, I removed the repeating paragraph.

Reply:
thank you!